microcosm_daemon/health_reporter.py

```diff
@@ -0,0 +1,57 @@
+import os
+from logging import getLogger
+
+from microcosm.api import defaults, typed
+
+from microcosm_daemon.error_policy import ExitError
+
+
+try:
+    import requests
+except ImportError:
+    requests = None  # type:ignore
```

Comment:
why is this code here? Is it because `requests` is not guaranteed to be loaded when running the daemon?

Reply:
Right, `requests` is one of the packages in the optional dependencies under `healthcheck`.